tests/test_cxl_host.py

```diff
@@ -146,7 +146,7 @@ async def send_util_and_check_host(host_client, util_client, cmd):
 @pytest.mark.asyncio
 async def test_cxl_host_manager_send_util_and_recv_host():
     host_port = BASE_TEST_PORT + pytest.PORT.TEST_1
-    util_port = BASE_TEST_PORT + pytest.PORT.TEST_1 + 50
+    util_port = BASE_TEST_PORT + pytest.PORT.TEST_1 + 10
```

Comment:
10 is not large enough number since it is likely (particularly in the future) that one test file has more than 10 test cases. i.e.
```
(BASE_TEST_PORT + pytest.PORT.TEST_1 + 10) == (BASE_TEST_PORT + pytest.PORT.TEST_10) == 9310
```
(seems like there is no TEST_0, so TEST_1 is likely 0).

Was there an issue with using 50?